cbmc/proofs/DeserializeSuback/Makefile

```diff
@@ -1,19 +1,53 @@
 ENTRY=DeserializeSuback
 
-BUFFER_SIZE = 100
+################################################################
+# Proof assumptions
 
-ABSTRACTIONS = \
-	--remove-function-body IotLog_Generic \
-	--remove-function-body _IotMqtt_RemoveSubscriptionByPacket \
+# Bound the number of subscriptions in a list (BOUND = MAX-1)
+SUBSCRIPTION_COUNT_MAX=2
+DEF += -DSUBSCRIPTION_COUNT_MAX=$(SUBSCRIPTION_COUNT_MAX)
 
-UNWINDING = \
-	--unwindset _decodeSubackStatus.0:$(BUFFER_SIZE) \
+# Bound the number of operations in a list (BOUND = MAX-1)
+OPERATION_COUNT_MAX=2
+DEF += -DOPERATION_COUNT_MAX=$(OPERATION_COUNT_MAX)
 
-OBJS = \
-	$(ENTRY)_harness.goto \
-	$(MQTT)/libraries/standard/mqtt/src/iot_mqtt_subscription.goto \
-	$(MQTT)/libraries/standard/mqtt/src/iot_mqtt_serialize.goto \
+#BUFFER_SIZE = 100
+BUFFER_SIZE = 15
+DEF += -DBUFFER_SIZE=$(BUFFER_SIZE)
 
-DEF = -DBUFFER_SIZE=$(BUFFER_SIZE)
+################################################################
+# Function omitted from the proof
+
+ABSTRACTIONS += --remove-function-body IotLog_Generic
+
+# Functions removed to make coverage calculation accurate
+# None
+
+# These defines have the effect of removing RemoveAllMatches from the
+# linear containers header so that our stub will be used.
+DEF += -DCBMC=1 -DCBMC_PROOF_DESERIALIZE_SUBACK=1
+
+################################################################
+# Loop unwinding
+
+LOOP += valid_IotMqttSubscriptionList.0:$(SUBSCRIPTION_COUNT_MAX)
+LOOP += valid_IotMqttOperationList.0:$(OPERATION_COUNT_MAX)
+
```

Comment:
remove extra line

Reply:
It separates loop unwinding for my helper functions from loop unwinding for the actual code.